src/addin/client-interfaces/addin-client-theme-settings.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Interface to specify the theme and mode to be applied to an add-in client.
+ */
+export interface AddinClientThemeSettings {
+
+  /**
+   * The theme configuration.
+   */
+  theme: string;
```

Comment:
Just checking - is this actually a string? or is it `default | modern`?

Reply:
This `theme` property should match the theme [name ](https://github.com/blackbaud/skyux/blob/514d6c37ae9ddfaa3d36031f9addaa83900f731a/libs/components/theme/src/lib/theming/theme.ts#L28) property on SkyTheme, which is a string.